README.md

```diff
@@ -628,3 +628,40 @@ or `#security` in MOJ Slack.
 
 1. **closed**
    The kilo has marked the case as closed.
+
+# How to upgrade Ruby 2.5.x to Ruby 2.7.x on local environment 
+
+1. Checkout the branch e.g master if the upgrade feature has merged, with ruby version defined as 2.7.2 in .ruby-version
```

Comment:
I think that there isn't a need to have "i.e. if the upgrade feature is merged" as once this PR is merged, this will be the case, and then that sentence becomes confusing.